code/game/objects/items/stacks/tiles/tile_types.dm

```diff
@@ -71,34 +71,28 @@
 	material = "wood"
 
 /obj/item/stack/tile/wood/proc/build(turf/S as turf)
-	if(istype(S,/turf/unsimulated))
+	if(istype(S,/turf/space))

```

Comment:
Am I to just snowflake every tile that has no air? eh. fuck it, this will do.

[also it just occured to me this can't even be built on space so this change was irrelevant but whatever]